src/Snppts/Snppts.csproj

```diff
@@ -6,7 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />

```

Comment:
I don't think this was intentional? Would you mind undoing this change? It's probably nothing, but I would hate if this messes with our CI/CD pipeline somehow ;)

Reply:
Done, must've been some VS magic.